docs/usage/self-hosted-configuration.md

```diff
@@ -1132,6 +1154,18 @@ The combinations of `requireConfig` and `onboarding` are:
 | `requireConfig=optional` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
 | `requireConfig=ignored`  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
 
+## s3Endpoint
+
+If set, Renovate will use this string as the `endpoint` when creating the AWS S3 client instance.
+
+## s3PathStyle
+
+If set, Renovate will enable `forcePathStyle` when creating the AWS S3 client instance.
+
+> Whether to force path-style URLs for S3 objects (e.g., `https://s3.amazonaws.com//` instead of `https://.s3.amazonaws.com/`)
+
+Source: [AWS S3 documentation - Interface BucketEndpointInputConfig](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/bucketendpointinputconfig.html)
+
```

Comment:
@Gabriel-Ladzaretti why is this necessary? Isn't it the same administrator who's configuring the S3?

Reply:
This is mainly used for s3 compatible solutions that dont support or are not configured to use `virtual-hosted–style` requests. It essentially alters the way the aws client constructs and resolves urls.
[path-style-access](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#path-style-access) 
e.g, `https://s3.amazonaws.com/mybucket/myfile`
vs
[virtual-hosted-style-access](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#virtual-hosted-style-access) 
e,g, `https://mybucket.s3.amazonaws.com/myfile`

i dont think there is another way to support these s3 instances.

Reply:
I use it for my Minio instance